documentation/staging/content/managing-domains/accessing-the-domain/remote-admin-console.md

```diff
@@ -21,7 +24,10 @@ The Remote Console is deployed as a standalone Java program, which can connect t
 You connect to the Remote Console and, when prompted, supply the WebLogic Server login credentials
```

Comment:
The Remote Console is not a standalone Java Program it is built using Oracle JET and packaged as an electron application.  The user does not need to know these details it is probably more important that we describe it as:
The WebLogic Remote Console provides an alternative WebLogic Server administration GUI that enables REST-based access to WebLogic management information.  The Remote Console allows to you to connect to multiple WebLogic Administration Servers, configure and monitor domains, and create and edit WebLogic Deploy Tooling models.

Reply:
I can not leave a note on the Note where it again specifies RC as a standalone Java program, please remove "standalone Java program from the notes below".    Also the Remote Console is normally ran in the users local desktop there are some cases where the users will invoke the Remote Console from their browser but it is not the common use case.

Reply:
so remove the word `Java` and I think that the existing description is fine.

Reply:
I have made the changes to remove the word `Java` and few other things as Monica suggested in [4666c27](https://github.com/oracle/weblogic-kubernetes-operator/pull/3525/commits/4666c2733e275004fbe02780d5abdfe5fde03ac7).